src/main/java/seedu/weme/model/RecordsManager.java

```diff
@@ -76,9 +76,7 @@ public void addDescription(Description description) {
     @Override
     public void addTags(Set<Tag> tags) {
         for (Tag tag: tags) {
-            if (!tag.toString().isEmpty()) {
-                tagRecords.add(tag.getTagName());
-            }
+            tagRecords.add(tag.getTagName());
```

Comment:
The original purpose is to not include empty tag in the records, otherwise there will be a blank line in the command prompt.

Reply:
Did you have a blank line when you tested? I removed it because I thought you can’t have a empty Tag due to it’s validations